mappings/net/minecraft/client/render/item/ItemRenderer.mapping

```diff
@@ -2,52 +2,73 @@ CLASS dkm net/minecraft/client/render/item/ItemRenderer
 	FIELD a ENCHANTMENT_GLINT_TEX Lqd;
 	FIELD b zOffset F
 	FIELD c WITHOUT_MODELS Ljava/util/Set;
-	FIELD d modelMap Ldez;
+	FIELD d models Ldez;
 	FIELD e textureManager Ldon;
 	FIELD f colorMap Lcqk;
-	METHOD a getModelMap ()Ldez;
+	METHOD a getModels ()Ldez;
 	METHOD a renderItemModel (IIZ)V
 		ARG 1 xPosition
 		ARG 2 yPosition
-	METHOD a renderItemInGui (Lahl;Laxd;II)V
+	METHOD a renderGuiItem (Lahl;Laxd;II)V
 		ARG 2 itemStack
-		ARG 3 x
+		ARG 3 xPosition
+		ARG 4 yPosition
 	METHOD a hasDepthInGui (Laxd;)Z
-	METHOD a renderItemWithPropertyOverrides (Laxd;II)V
+	METHOD a renderGuiItemIcon (Laxd;II)V
 		ARG 1 stack
 		ARG 2 xPosition
```

Comment:
`xPosition` and just `x` seem to be inconsistently used in the parameters here

Reply:
Yeah, check the rest of the class to ensure they all make sense. `renderGuiItemModel` uses `x` and `y` for example